content/en/blog/_posts/2025-XX-XX-kubectl-kuberc-beta.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: blog
+title: "Kubernetes v1.34: User preferences (kuberc) are available for testing in kubectl 1.34"
+draft: true
+slug: kubernetes-v1-34-kubectl-kuberc-beta
+Author: >
+  [Maciej Szulik](https://github.com/soltysh) (Defense Unicorns, Inc.)
+---
+
+Have you ever wished you could enable [interactive delete](https://kep.k8s.io/3895),
+by default, in `kubectl`? Or maybe, you'd like to have custom aliases defined,
+but not necessarily [generate hundreds of them manually](https://github.com/ahmetb/kubectl-aliases)?
+Look no further. [SIG-CLI](https://git.k8s.io/community/sig-cli/)
+has been working hard to add [user preferences to kubectl](https://kep.k8s.io/3104),
+and we are happy to announce that this functionality is reaching beta as part
+of the Kubernetes v1.34 release.
+
+## How it works
+
+A full description of this functionality is available [in our official documentation](/docs/reference/kubectl/kuberc/),
+but this blog post will answer both of the questions from the beginning of this
```

Comment:
```suggestion
but this blog post will answer both questions from the beginning of this
```

Reply:
Honestly, that was my original text, but after initial offline review, I added "of the" :wink: From what I'm looking both are correct, with the one I have being more formal. So I'd leave it as is. But thank you for your comment :smile:

Reply:
ok sounds good!